homeassistant/components/lamarzocco/switch.py

```diff
@@ -67,12 +60,20 @@ async def async_setup_entry(
     """Set up switch entities and services."""
 
     coordinator = hass.data[DOMAIN][config_entry.entry_id]
-    async_add_entities(
+    entities: list[SwitchEntity] = []
+    entities.extend(
         LaMarzoccoSwitchEntity(coordinator, description)
         for description in ENTITIES
         if description.supported_fn(coordinator)
     )
 
+    entities.extend(
+        LaMarzoccoAutoOnOffSwitchEntity(coordinator, wake_up_sleep_entry_id)
+        for wake_up_sleep_entry_id in coordinator.device.config.wake_up_sleep_entries
+    )
```

Comment:
Followup

Reply:
I thought about that, but it's weird for an entity to be gone one month to make a return

Reply:
I mean people still have to adapt, we are now at the start of the release, so if we take it through we can make it

Reply:
Let's focus on the other things first, if those are fine we can do this